front/src/common/BootstrapSNCF/NavBarSNCF.tsx

```diff
@@ -47,11 +47,11 @@ const LegacyNavBarSNCF = ({ appName, showLogoWithName }: Props) => {
   }, [deploymentSettings, showLogoWithName]);
 
   const userDropdownTitle = (
-    <div className={cx({ 'impersonated-user': impersonatedUser })}>
+    <div className={cx('user-dropdown', { 'impersonated-user': impersonatedUser })}>
       {impersonatedUser ? (
-        <Hubot size="lg" className="mr-xl-2" />
+        <Hubot size="sm" className="mr-xl-2" />
       ) : (
-        <Person variant="fill" size="lg" className="mr-xl-2" />
+        <Person variant="base" size="sm" className="mr-xl-2" />
```

Comment:
There isn't any mockup for smaller screens but maybe we should keep the 8px margin all the time (same for `Hubot)` or better we could put a `ml-2` for the username `span` instead.
```suggestion
        <Person variant="base" size="sm" className="mr-2" />
```

Reply:
We will see what changes need to be made once we have a mockup.

Reply:
Keep the same spacing between elements for smaller width. This is going to be replaced soon, so we should not spend too much time making it perfect.